tests/docker-compose.yml

```diff
@@ -29,6 +29,13 @@ services:
       BEANSTALK_HOST: beanstalk
       GEARMAN_HOST: gearmand
       COMPOSER_ALLOW_SUPERUSER: 1
+      AWS_SQS_ENABLED: ${AWS_SQS_ENABLED}
+      AWS_KEY: ${AWS_KEY}
+      AWS_SECRET: ${AWS_SECRET}
+      AWS_REGION: ${AWS_REGION}
+      AWS_SQS_URL: ${AWS_SQS_URL}
+      AWS_SQS_FIFO_URL: ${AWS_SQS_FIFO_URL}
+      AWS_SQS_FIFO_MESSAGE_GROUP_ID: ${AWS_SQS_FIFO_MESSAGE_GROUP_ID}
```

Comment:
`AWS_SQS_FIFO_URL` and `AWS_SQS_FIFO_MESSAGE_GROUP_ID` are not used yet.

Reply:
Yes, but they are added in #332 
I'll remove them